output/slide-deck.qmd

```diff
@@ -530,62 +428,85 @@ git config --list --global
 Paste in terminal with `Shift-Insert`.
 :::
 :::
-:::
 
 ::: aside
-\*: You will only have to do this once
+\*: You will only have to do this once (by computer)
 :::
 
 ::: notes
 The one and only time that we will open the Terminal. Why doesn't Rstudio
 provide GUI functionality for this? I don't know...
 
-Mind the double quotes; type in your name or GitHub user name. This is what will
-appear in your repo "history" when you "commit" or create versions
+Mind the double quotes; type in your name or (preferably) GitHub user name. This
+is what will appear in your repo "history" when you "commit" or create versions,
+e.g.:
+
+`git config --global user.name "DaniMori"`
 
 Type in your email (also mind the double quotes). Better use "User name" at
-"users.noreply.github.com"
+"users.noreply.github.com" (for obfuscating actual email from public exposure),
+e.g.:
+
+git config --global user.email "DaniMori\@users.noreply.github.com"
 :::
 
-## Let's make some changes
+## Let's make some changes {.smaller}
+
+<br>
 
--   Delete unnecessary folders / files
+-   Delete unnecessary folders
 
 ::: {.nonincremental .fragment}
--   Edit `README.md`:
+-   Edit "README.Rmd":
 
-    -   Change \<repository-name\> by your repo (lines 2 & 113)
+    -   Change `<repository-name>` by your repo (lines 12-14)
 
-    -   Change \<repository-url\> by yours (line 66)
+    -   Change `<username>` by your GitHub user name (lines 16-18)
 
-    -   Edit "License" section (lines 8-32)
+    -   Edit description in section "Repository" (lines 23-24)
 
-    -   Edit "description" (line 4)
+    -   Delete description of unused folders (lines 158-181)
 :::
 
+-   Knit "README.md"
```

Comment:
Might not know where the knit button is - obviously we'll tell them, but could also add to text: "Knit README.md (blue yarn button at top of script)

Reply:
Will it be enough with demonstrating it beforehand?

Also, we can add a screenshot (I'll add your suggestion for now)

Reply:
Addressed in ea19cfa84c3e10d14802c15c23787d6979d8b510